template/web/src/components/FallbackImage/FallbackImage.module.css

```diff
@@ -0,0 +1,4 @@
+.FallbackImageIcon {
+  width: clamp(0%, 50%, 100%);
```

Comment:
Can we use Tailwind here if possible?

Reply:
Just checked and `Clamp` utility is still not implemented https://github.com/tailwindlabs/tailwindcss/discussions/10647

Reply:
Do we need clamp or can we use something else?

Reply:
Good question. I used it to have image-fallback svg stretch with the size of the container. I'll see if I can find a simpler alternative